internal/hardware/validator_test.go

```diff
@@ -650,14 +649,16 @@ var _ = Describe("Disk eligibility", func() {
 		testDisk.SizeBytes = tooSmallSize
 		testDisk.InstallationEligibility.NotEligibleReasons = []string{}
 
-		requirements, err := hwvalidator.GetClusterHostRequirements(ctx, &cluster, &host)
-		Expect(err).ToNot(HaveOccurred())
-		minSizeBytes := conversions.GbToBytes(requirements.Total.DiskSizeGb)
+		// The validator now uses preciseHumanizeBytes, so we need to match that format
+		// For 99999999999 bytes (100GB-1), preciseHumanizeBytes shows "100.0 GB"
+		// For 100000000000 bytes (100GB), preciseHumanizeBytes shows "100 GB"
+		diskSizeStr := "100.0 GB" // 99999999999 bytes shows as 100.0 GB with precise function
+		reqSizeStr := "100 GB"    // 100000000000 bytes shows as 100 GB with precise function
 
 		expectedMsg := fmt.Sprintf(
 			tooSmallDiskTemplate,
-			humanize.Bytes(uint64(testDisk.SizeBytes)),
-			humanize.Bytes(uint64(minSizeBytes)),
+			diskSizeStr,
+			reqSizeStr,
```

Comment:
it's still not clear from user point of view, they will see: `disk only has 100.0 GB but requires 100GB`, right?